### PR TITLE
Bug create activity cluster ref no #230

### LINKED ISSUE #230: Add cluster activity CO ref # should be a dropdown not a text field
Its a text field in adding CO, but for cluster activity it should be a dropdown (basically user is selecting a cluster objective)

Options should be for the cluster objectives in this cluster and response plan. Backend should validate the same.

Tried few things, manually entering the CO # ref as well... get errors: {"cluster_objective":["Invalid pk \"443\" - object does not exist."]}

### PR DESCRIPTION
**Pull request by ErinOC**
This PR resolves #230: Add cluster activity CO ref # should be a dropdown not a text field 

After selecting a cluster, the cluster objective ref no. dropdown will display the cluster objective reference numbers specific to that cluster and ID. The dropdown is disabled when no cluster is selected.

- [x ] Polymer

- [ ] Polymer test cases

### CODE REVIEW

File: polymer/src/elements/cluster-reporting/response-parameters/clusters/activities/creation-modal.html [hunk 1]
```diff
@@ -232,21 +250,76 @@
...
         return App.Endpoints.responseParametersClusterObjectives(responsePlanID);
       },
 
+      _computeObjectivesUrl: function(responsePlanID) {
+        return App.Endpoints.responseParametersClusterObjectives(responsePlanID);
+      },
+
+      _isObjectivesDisabled: function(clusterID) {
+        if (clusterID) {
+          this.getObjectivesByClusterID();
```
Comment (rkrupinski) [inline polymer/src/elements/cluster-reporting/response-parameters/clusters/activities/creation-modal.html]: Please, don't fetch/set/update stuff within computed props. Use observers instead.

Comment (rkrupinski) [reply polymer/src/elements/cluster-reporting/response-parameters/clusters/activities/creation-modal.html]: @ErinOC ^^

Comment (ErinOC) [reply polymer/src/elements/cluster-reporting/response-parameters/clusters/activities/creation-modal.html]: I'm under another deadline currently, but I'll fix this on Thursday.

Comment (rkrupinski) [reply polymer/src/elements/cluster-reporting/response-parameters/clusters/activities/creation-modal.html]: Cool, just wanted to know whether we keep finger on the pulse here.

Comment (ErinOC) [reply polymer/src/elements/cluster-reporting/response-parameters/clusters/activities/creation-modal.html]: I didn't get to this today, unfortunately. I haven't forgotten about it.

**Pull request merged**